### PR TITLE
Create GetUserAccount endpoint with CreateUserAccount function(Want to push #79)

### LINKED ISSUE #70: Create GetUserAccount endpoint with CreateUserAccount function
from #57 

https://github.com/ice-cream-the-fat-cat/ice-cream-backend/issues/57#issuecomment-911266359

Create endpoint
- GetUserAccount{:userid}

and, if there are no data in UserAccount table, endpoint call the following function
- CreateUserAccount

### PR DESCRIPTION
**Pull request by yukari-n-erb**
## Description

Add GetUserAccount endpoint with CreateUserAccount function to save user coin balance and unlock flowers.

## Closes issue(s)

#70 

## Screenshots

- Send 2 request, result and database data
GET /api/v1/users/xxxxxx

Screen Shot 2021-09-04 at 11 45 48
Screen Shot 2021-09-04 at 11 45 53

GET /api/v1/users/xxxxxx1
Screen Shot 2021-09-04 at 11 46 05
Screen Shot 2021-09-04 at 11 46 08

- Finally, I resend the following request, and I can get existing dats withoutacreating a new user.

GET /api/v1/users/xxxxxx

Screen Shot 2021-09-04 at 11 46 18

Screen Shot 2021-09-04 at 11 58 00

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [x] I have tested this code

## Other comments
This PR's users model has already FireBaseUserId column.
(ref : Feature fix fire base user #79 )

### CODE REVIEW

**Comment by yukari-n-erb** [comment]
memo : How can I add pass userFireBaseID and return ...

Screen Shot 2021-09-02 at 21 48 10

File: controllers/v1/users/users_controller.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+func GetUserByUserId(createUserId interface{}) (users_models.Users, error) {
+	ctx := mongo_connection.ContextForMongo()
+	client := mongo_connection.MongoConnection(ctx)
+
+	defer client.Disconnect(ctx)
+	collection := mongo_connection.MongoCollection(client, "users")
+
+	var result users_models.Users
+	err := collection.FindOne(ctx, bson.D{
+		primitive.E{Key: "_id", Value: createUserId},
```
Comment (kome12) [inline controllers/v1/users/users_controller.go:40]: @yukari-n-erb Is this supposed to look at MongoDB "_id" or the fireBaseUserId?

**Comment by yukari-n-erb** [comment]
memo : 
after merge #79, I will merge this PR just in case to avoid conflict

File: controllers/v1/users/users_controller.go (hunk 1 above)
Comment (yukari-n-erb) [reply controllers/v1/users/users_controller.go]: Thank you! you are right so I fix it.

**Review by kome12 — APPROVED** [review]
LGTM!

**Comment by kome12** [comment]
@yukari-n-erb I'll merge this for now because I need this change to work on the next few issues!

**Pull request merged**